content/accessibility/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Accessibility
+---
+
+import {Grid, Flex, Box, Link, Text, Label} from '@primer/components'
+
+<Grid gridTemplateColumns="1fr" gridGap={4}>
+
+
+<Grid gridTemplateColumns="1fr" gridGap={4}>
+    <h2>General</h2>
+    <div>
+        <Link href="/design/content/accessibility/accessibility-at-github" fontWeight="bold">Accessibility at GitHub</Link>
+        <Box as="p">Get started with accessibility at GitHub.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/guidelines" fontWeight="bold">Guidelines</Link>
+        <Box as="p">Basic guidelines for designers when creating or updating features.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/tools" fontWeight="bold">Tools</Link>
+        <Box as="p">A selection of tools to help designers create accessible designs.</Box>
+    </div>
+</Grid>
+
+
+<Grid gridTemplateColumns="1fr" gridGap={4}>
+    <h2>Specific</h2>
+    <div>
+        <Link href="/design/content/accessibility/alternative-text-for-images" fontWeight="bold">Alternative text for images</Link>
+        <Box as="p">Alternative text on images is what allows assitstive techonology like screen readers to understand the purpose of an image on a page or allow them to skip it because it's purely decorative. </Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/announcements" fontWeight="bold">Assistive technology announcements</Link>
+        <Box as="p">Things like toasts and status messages are visually communicated to GitHub users and making sure those announcements are read via assitstive technology allow users with low or no vision to get that status information.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/button-purpose" fontWeight="bold">Describing a button’s purpose</Link>
+        <Box as="p">Labeling buttons properly let's users know what will happen when they activate the control, lessens errors, and levels up confidence.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/focus-management" fontWeight="bold">Focus management</Link>
+        <Box as="p">Managing focus within a page or application is essential for users to successfully navigate, complete actions, and understand where they are.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/headings" fontWeight="bold">Headings</Link>
+        <Box as="p">Headings play a critical role in communicating the structure of a page. Find out why they're critical and how to create an accessible hierarchy in your pages.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/links" fontWeight="bold">Links</Link>
+        <Box as="p">Links help us navigate a website. Learn how to style links appropriately to keep them usable by all.</Box>
+    </div>
+    <div>
+        <Link href="/design/content/accessibility/semantic-html" fontWeight="bold">Semantic HTML</Link>
```

Comment:
Isn't semantic HTML general?

Reply:
Somewhat? But it's specific to building accessible components. It's a specific topic about accessibility where the others are little collections of multiple things.